tests/lca/imagebasedupgrade/mgmt/negative/tests/ibu-invalid-next-stage.go

```diff
@@ -0,0 +1,76 @@
+package negative_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/lca"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedupgrade/internal/nodestate"
+	. "github.com/openshift-kni/eco-gotests/tests/lca/imagebasedupgrade/mgmt/internal/mgmtinittools"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedupgrade/mgmt/negative/internal/tsparams"
+	"golang.org/x/exp/slices"
+)
+
+var _ = Describe(
+	"Patching ibu stage from Idle to Rollback",
```

Comment:
Make it more generic as this test file will contain the other invalid transitions as well

Reply:
Done.